src/components/alert/alert.svelte

```diff
@@ -21,6 +21,7 @@
   export let isToast = false
   export let hasActions = $$slots.actions
   export let hasContentAfter = $$slots['content-after']
+  export let hideIcon = false
```

Comment:
will it show on the story?

Reply:
Yup, props are automatically in the story :smile: